src/Components/Navbar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import { BsChevronLeft } from 'react-icons/bs';
+import '../styles/navbar.css';
+
+const Navbar = () => {
+  const history = useHistory();
+  const location = useLocation();
+
+  return (
+    <nav>
+
+      <div className="location-path">
+        {(location.pathname !== '/') ? <button className="back" type="button" onClick={history.goBack}><BsChevronLeft className="fas fa-chevron-left"> </BsChevronLeft></button> : <span className="back">Countries</span>}
```

Comment:
- [ ] Kindly look at the image below. When I go from the **`root`** page to the **`country`** page, and then back to the **`root`** the loading image continues to persists despite having my data on the **`root`** page already. Please fix this.

![redwing-loading](https://user-images.githubusercontent.com/36542986/142625536-c44c0dec-f49b-4ccc-8388-68d049ebd53b.jpg)